mathesar_ui/src/components/cell-fabric/data-types/components/number/NumberCell.svelte

```diff
@@ -38,6 +38,25 @@
     maximumFractionDigits,
   });
 
+<<<<<<< HEAD
```

Comment:
Lines that begin with `<<<<<<<` or `=======` or `>>>>>>>` should not be in the diff. Those are from git merge conflicts.